tests/runfiles/linux.run

```diff
@@ -756,7 +756,8 @@ tags = ['functional', 'refquota']
 
 [tests/functional/refreserv]
 tests = ['refreserv_001_pos', 'refreserv_002_pos', 'refreserv_003_pos',
-    'refreserv_004_pos', 'refreserv_005_pos']
+    'refreserv_004_pos', 'refreserv_005_pos', 'refreserv_multi_raidz',
+    'refreserv_raidz']
```

Comment:
The new test cases will also need to be added to the `Makefile.am` in the relevant subdirectory.

Reply:
ok

Reply:
Fixed in https://github.com/zfsonlinux/zfs/pull/8973

Similar issue that is specific to this PR has also been fixed for `cli_root/zfs_create/Makefile.am`